### PR TITLE
Disable TEMPLATE from sidebar of Korean site

### PR DESCRIPTION
**Pull request by seokho-son**
Signed-off-by: Seokho Son <shsongist@gmail.com>

### Describe your changes
Disable TEMPLATE from sidebar of Korean site.
Status: Completed makes the TEMPLATE item shown in the sidebar 

### Related issue number or link (ex: `resolves #issue-number`)
N/A

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco).

### CODE REVIEW

File: content/ko/_TEMPLATE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: 용어 정의 템플릿
-status: Completed
+status: Feedback Appreciated
```
Comment (jihoon-seo) [inline content/ko/_TEMPLATE.md:3]: https://github.com/cncf/glossary/pull/1232#issuecomment-1217602013 에 쓰신 내용과 같이
`content/ko/_TEMPLATE.md` 를 삭제할 수도 있고,

또는
https://github.com/cncf/glossary/pull/1232/files#r965527390 에 쓰신 것 처럼 할 수도 있겠습니다. 😊

```suggestion
status: Feedback Appreciated (가이드: status 값은 영어 원문과 동일하게 유지)
```

**Review by jihoon-seo — APPROVED** [review]
LGTM!

**Comment by seokho-son** [comment]
@jihoon-seo I updated PR, I think some characters for the status value are not acceptable.
changed to use `#` comment

**Comment by seokho-son** [comment]
PTAL @yunkon-kim

**Review by yunkon-kim — APPROVED** [review]
LGTM too :-)

**Pull request merged**